test/common/upstream/load_stats_reporter_test.cc

```diff
@@ -74,7 +76,9 @@ class LoadStatsReporterTest : public testing::Test {
   Event::TimerCb response_timer_cb_;
   Grpc::MockAsyncStream async_stream_;
   Grpc::MockAsyncClient* async_client_;
-  MockMonotonicTimeSource time_source_;
+  ProdSystemTimeSource system_time_source_;
```

Comment:
Why is this prod? Won't your recent fix_format check complain?

Reply:
It whitelists test/*

Reply:
It whitelists test/*

Reply:
OK, but why not use a mock alternative all things being equal when adding new entries here?

Reply:
I think that would change behavior if it were called. Current status is that real time is used everywhere in tests where there is not already an explicit mock. These can be cleaned up one time is injectable.

Note in particular that dispatcher uses real time for timers unconditionally, though I have some pushed WIP and unpushed changes I am working that add simulated timers.

Reply:
In any event it doesn't affect this test, and in an upcoming #4257 I'll effectively be making this semantic change anyway, so I went ahead and made the change you suggested.